setup.py

```diff
@@ -7,36 +7,34 @@
 __version__ = VERSION
 __versionstr__ = ".".join(map(str, VERSION))
 
-f = open(join(dirname(__file__), "README"))
-long_description = f.read().strip()
-f.close()
+with open(join(dirname(__file__), "README")) as f:
+    long_description = f.read().strip()
 
 install_requires = ["urllib3>=1.21.1"]
 tests_require = [
     "requests>=2.0.0, <3.0.0",
     "nose",
     "coverage",
     "mock",
-    "pyaml",
+    "pyyaml",
```

Comment:
`pyaml` is an unrelated package to what we actually want, which is `PyYAML`. Just so happened that `pyaml` had `PyYAML` as a dependency!

Reply:
wow, thanks for the correction, I never checked